pylink/library.py

```diff
@@ -202,6 +218,8 @@ def find_library_linux(cls):
 
             for fname in fnames:
                 fpath = os.path.join(directory_name, fname)
+                if not cls.can_load_library(fpath):
```

Comment:
Should this check actually occur after the OS check?

Reply:
Yes, should only be checked on 64 bit OSes. Changed